django/views/templates/technical_500.html

```diff
@@ -132,6 +132,12 @@ <h1>{% if exception_type %}{{ exception_type }}{% else %}Report{% endif %}
       <th>Exception Location:</th>
       <td><span class="fname">{{ lastframe.filename }}</span>, line {{ lastframe.lineno }}, in {{ lastframe.function }}</td>
     </tr>
+{% endif %}
+{% if raising_view_name %}
+    <tr>
+      <th>Raised during:</th>
+      <td>{{ raising_view_name }}</td>
```

Comment:
"Raised during" seems odd to me. Would "View name" not suffice?

```suggestion
{% if view_name %}
    <tr>
      <th>View name:</th>
      <td>{{ view_name }}</td>
```

Reply:
I would leave `raising_view_name` to keep it sync with `technical_404.html`.

Reply:
The reason I proposed `Raised during` or `During view` or whatever is for a sort of consistency with the 404 response, which uses `Raised by`

`Raised by` is correct enough in that scenario because generally speaking a 404 would be raised _from_ the view layer.
Raised _by_ is incorrect for the 500 response though, because in the general sense it can be anywhere down/upstream of the view itself (e.g. could be anything, like a `ZeroDivisionErrror` down in the bowels of a lib being used)

`Raised during` and `During view` were my ideas to consolidate the messaging across both technical pages to be 'correct enough' that it doesn't matter overly.

YMMV.

Reply:
Ah, ok. Makes sense I guess. Ignore me 🙂

Reply:
> Makes sense I guess. Ignore me 🙂

No :P There's no reason _not_ to consider alternative wordings, if we can find one that fits both scenarios and makes sense in the general case, that works for everyone, perfect. I'll happily to defer to the general consensus on the best wording, because whilst I did attempt to consider it, I'm only a data point of one.